pages/_/marketing-candidate/download.js

```diff
@@ -10,99 +10,156 @@ import WebsitePrototypeHeader from "~/components/core/NewWebsitePrototypeHeader"
-    h1 {
-      font-size: 1.953rem;
-      padding: 0px 0px 16px 0px;
-      line-height: 1.25;
-    }
-    h2 {
-      font-size: 1.25rem;
-      padding: 0px 0px 8px 0px;
-      line-height: 1.5;
-    }
-    h3 {
-      font-size: 1rem;
-      padding: 0px 0px 8px 0px;
-      line-height: 1.5;
-      color: ${Constants.system.moonstone};
-    }
-    p {
-      font-size: 0.78rem;
-    }
-    img {
-      width: 80%;
-      hight: auto;
-    }
+    font-size: 1.953rem;
+    padding: 0px 0px 16px 0px;
+    line-height: 1.25;
   }
 `;
 
-const STYLES_APP = css`
-  display: flex;
-  flex-direction: row;
-  align-self: center;
+const STYLES_H2 = css`
+  font-size: 1.25rem;
+  line-height: 1.25;
+  padding: 0px 0px 32px 0px;
+  width: 100%;
+  color: ${Constants.system.darkGray};
+  @media (max-width: ${Constants.sizes.mobile}px) {
+    font-size: 1.25rem;
+    padding: 0px 0px 8px 0px;
+    line-height: 1.5;
+  }
+`;
 
+const STYLES_H3 = css`
+  font-size: 1.563em;
+  line-height: 1.5;
+  padding: 0px 0px 16px 0px;
+  color: ${Constants.system.darkGray};
   @media (max-width: ${Constants.sizes.mobile}px) {
-    flex-direction: column;
+    font-size: 1rem;
+    padding: 0px 0px 8px 0px;
+    line-height: 1.5;
+    color: ${Constants.system.moonstone};
   }
 `;
 
-const STYLES_EXTENSTION = css`
-  display: flex;
-  flex-direction: row;
-  align-self: center;
+const STYLES_P = css`
+  font-size: 1rem;
+  color: ${Constants.system.black};
   @media (max-width: ${Constants.sizes.mobile}px) {
-    flex-direction: column;
+    font-size: 0.78rem;
   }
 `;
 
-const STYLES_WRAPPER_TEXT = css`
-  width: 40%;
-  align-self: center;
-  padding: 0 50px;
+const STYLES_A = css`
+  :link {
+    color: ${Constants.system.darkGray};
+    background-color: transparent;
+    text-decoration: none;
+  }
+  :hover {
+    color: ${Constants.system.pitchBlack};
+    background-color: transparent;
+    text-decoration: none;
+  }
+  :active {
+    color: yellow;
+    background-color: transparent;
+    text-decoration: none;
+  }
   @media (max-width: ${Constants.sizes.mobile}px) {
-    flex-direction: column;
-    width: 100%;
-    padding: 0 10px;
   }
 `;
 
-const STYLES_BROWSER_WINDOW = css`
-  background: #ffffff;
-  box-shadow: 0px 10px 50px 20px rgba(0, 0, 0, 0.1);
-  border-radius: 10px;
-  align-self: center;
+const STYLES_IMG = css`
+  width: 100%;
+  :hover {
+  }
+  @media (max-width: ${Constants.sizes.mobile}px) {
+  }
 `;
 
 const STYLES_ANNOTATION = css`
+  padding-top: 7px;
   font-size: 12px;
   color: #646464;
 `;
 
+const STYLES_BUTTON_PRIMARY = css`
+  box-sizing: border-box;
+  border-radius: 2px;
+  outline: 0;
+  border: 0;
+  min-height: 40px;
+  padding: 6px 24px 6px 24px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  letter-spacing: 0.2px;
+  font-family: ${Constants.font.semiBold};
+  transition: 200ms ease all;
+  user-select: none;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.15);
```

Comment:
Careful will, don't de-dupe styles in the same class. `box-shadow` is here twice.